Makefile

```diff
@@ -36,6 +36,9 @@ generate:
 	find deploy/crds -name '*.yaml' | xargs go run ./cmd/trim-crd
 	go run github.com/ahmetb/gen-crd-api-reference-docs -api-dir ./pkg/apis -config docs/api/config.json -template-dir docs/api/template -out-file docs/api/index.html
 
+generate-operator-yaml:
+	go run github.com/kubernetes-sigs/kustomize build ./deploy > operator.yaml
```

Comment:
I like the idea, but could we place the file in `build/_output/operator.yaml` so we keep all our git-ignored build artifacts in one directory tree?

Reply:
For sure! That'd work.